management-gateway/src/test/java/org/frankframework/management/bus/ResourceNotFoundExceptionTest.java

```diff
@@ -0,0 +1,31 @@
+package org.frankframework.management.bus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.logging.log4j.Level;
+import org.frankframework.core.IbisException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class ResourceNotFoundExceptionTest {
+
+	private static final String RESOURCE_NOT_FOUND_EXCEPTION_MESSAGE = "outer exception";
+
+	public static List<?> data() {
+		return Arrays.asList(new Object[][]{
+				{"", Level.WARN, null},
+				{": cannot configure", Level.ERROR, new IbisException("cannot configure")},
+				{": cannot configure: (IllegalStateException) something is wrong", Level.ERROR, new IbisException("cannot configure", new IllegalStateException("something is wrong"))},
+		});
+	}
+
+	@ParameterizedTest
+	@MethodSource("data")
+	public void testExceptionMessage(String expectedLogMessage, Level expectedLogLevel, Exception innerException) {
```

Comment:
Perhaps I'm overlooking something, but where do you use the `expectedLogLevel`?

Reply:
It is somehow needed. I indeed used another class as an example but when removing the parameter, it gives a ParameterResolutionException

Reply:
That is because the `data` method, where your test-data comes from, gives some test-data including this value.

You need to remove it from that `data` method in your test-class as well.